kde-frameworks/kio/kio-5.109.0.ebuild

```diff
@@ -19,8 +19,6 @@ IUSE="acl +handbook kerberos +kwallet X"
 RESTRICT="test"
 
 RDEPEND="
-	dev-libs/libxml2
-	dev-libs/libxslt
```

Comment:
This also turns out not to be that easy. I traced it back to https://github.com/gentoo/kde/commit/cd9341e9c5693512297c3cfc1714523096bd7771 and https://github.com/gentoo/kde/commit/0e7d50740b1bf730177aa4b6c762a5d72ece08f8, both of which still hold true today:

```
$ lddtree /usr/lib64/qt5/plugins/kf5/kio/kio_help.so
[...]
    libKF5DocTools.so.5 => /usr/lib64/libKF5DocTools.so.5
        libxslt.so.1 => /usr/lib64/libxslt.so.1
    libxml2.so.2 => /usr/lib64/libxml2.so.2
        liblzma.so.5 => /lib64/liblzma.so.5
        libicuuc.so.73 => /usr/lib64/libicuuc.so.73
            libicudata.so.73 => /usr/lib64/libicudata.so.73
    libexslt.so.0 => /usr/lib64/libexslt.so.0
        libgcrypt.so.20 => /usr/lib64/libgcrypt.so.20
            libgpg-error.so.0 => /usr/lib64/libgpg-error.so.0
```
```
$ lddtree /usr/lib64/qt5/plugins/kf5/kio/kio_ghelp.so
[...]
    libKF5DocTools.so.5 => /usr/lib64/libKF5DocTools.so.5
        libxslt.so.1 => /usr/lib64/libxslt.so.1
    libxml2.so.2 => /usr/lib64/libxml2.so.2
        liblzma.so.5 => /lib64/liblzma.so.5
        libicuuc.so.73 => /usr/lib64/libicuuc.so.73
            libicudata.so.73 => /usr/lib64/libicudata.so.73
    libexslt.so.0 => /usr/lib64/libexslt.so.0
        libgcrypt.so.20 => /usr/lib64/libgcrypt.so.20
            libgpg-error.so.0 => /usr/lib64/libgpg-error.so.0
```

However, we could rightfully assume that `dev-libs/libxml2` and `dev-libs/libxslt` are being wrongfully unconditionally depended on here - and that's right after looking at https://invent.kde.org/frameworks/kio/-/blob/kf5/src/kioworkers/help/CMakeLists.txt.  These deps had been there since the very beginning of that ebuild (https://github.com/gentoo/kde/commit/7b5c719ed77b8ae9e123b6bc437d5a9fc54bfbf8).